tests/test_dag_utils_status.py

```diff
@@ -105,6 +111,7 @@ def exception():
 
     expected = {
         "test": State.FAILED,
+        "test_register": State.NONE,
```

Comment:
This is actually not ideal because the job should be registered regardless on the success state of the parent. In practice, this is fine because on failure, the StatuspageOperator will execute and update the page.

Reply:
put this in a comment?